hbase-it/src/test/java/org/apache/hadoop/hbase/DistributedHBaseCluster.java

```diff
@@ -126,6 +126,20 @@ public void waitForRegionServerToStop(ServerName serverName, long timeout) throw
     waitForServiceToStop(ServiceType.HBASE_REGIONSERVER, serverName, timeout);
   }
 
+  @Override
+  public void suspendRegionServer(ServerName serverName) throws IOException {
+    LOG.info("Suspend RS: " + serverName.getServerName());
```

Comment:
Please use parametrized ({}) logging **everywhere.**

Reply:
Thanks, I'll change this in every affected class.